moves/moves.go

```diff
@@ -20,12 +20,17 @@ type Roll interface {
 //grabs the nanosecond as a seed
 func seed() int64 {
 	now := time.Now()
-	return now.Unix()
+	return now.UnixNano()
 }
 
 func (m Move) Roll(bonus int) int {
+	//roll the first d6, +1 since Intn is 0 based
 	rand.Seed(seed())
-	return ((rand.Intn(10)) + 2) + bonus
+	d1 := (rand.Intn(6) + 1)
```

Comment:
I'm thinking: now that we have multiple rolls, what if the seed function becomes `die`, and returns `int` instead of int64?

Super quick concept:

```
func die(dieType int) int {
    // seed time code
    now := time.Now()
    rand.Seed(now.UnixNano())
    return rand(Intn(dieType) +1)
}
```

and then we can use the multiple dice more fluidly and more DRY in the actual Roll.

What do you think?